### ISSUE TITLE
Deploys on DEV/STAGING failing intermittently with "(98)Address already in use: AH00072: make_sock: could not bind to address [::]:80"

### ISSUE THREAD

**Issue by ElijahLynn** [department-of-veterans-affairs/va.gov-cms#5862]
### Description: 
Deploys are failing intermittently with:

> fatal: [ip-10-247-35-123.us-gov-west-1.compute.internal]: FAILED! => changed=false 
> 11:59:12    msg: |-
> 11:59:12      (98)Address already in use: AH00072: make_sock: could not bind to address [::]:80
> 11:59:12      (98)Address already in use: AH00072: make_sock: could not bind to address 0.0.0.0:80
> 11:59:12      no listening sockets available, shutting down
> 11:59:12      AH00015: Unable to open logs

![image](https://user-images.githubusercontent.com/1504756/125685762-695c2e78-ba25-4df8-a8b1-4802216baefc.png)

They usually succeed if we replay the job. 

### Tasks:
- [ ] Figure out how to reproduce
- [ ] Fix issue
  - [x] Configure `httpd` config in build stage to only Listen on IPv4 
  https://github.com/department-of-veterans-affairs/devops/pull/9604
  - [ ] Test branch on DEV
  - [ ] Reset branch on Jenkins build job

### CMS Team

- [x] `Core Application Team`
- [ ] `Product Support Team`

**Comment by ElijahLynn** [comment]
Caught this on a deploy to STAGING just now. 

![image](https://user-images.githubusercontent.com/1504756/125869207-10fae698-fb59-4e85-bc31-f0aec399d5d1.png)

**Comment by ElijahLynn** [comment]
I tried a restart again many minutes later and it worked fine. Very perplexed right now.

**Comment by ElijahLynn** [comment]
Asked on ServerFault here > https://serverfault.com/questions/1070051/intermittent-address-already-in-use-ah00072-make-sock-could-not-bind-to-addr

**Comment by ElijahLynn** [comment]
From a [Slack conversation](https://dsva.slack.com/archives/CT4GZBM8F/p1626719708311800): 


Elijah: httpd was hung, couldn't even stop it. 


![image](https://user-images.githubusercontent.com/1504756/126261119-2ffe6148-548d-4a48-bb7e-a13e650eec9c.png)

`sudo service httpd status` 

> httpd is stopped

https://files.slack.com/files-pri/T03FECE8V-F028QEYJ2BW/image.png

Elijah: Oh, but httpd with IPV6 is still listening somehow

Elijah: I `kill <pid>`  (no -9) and it took out the ipv6 listeners and allowed apache to start again. So, an easy route is to do that.

Adrian: seems like not listening on ipv6 at all would be the easiest path forward?

Elijah: Yeah, I will do that now. Need to get this knocked out.

**Comment by ElijahLynn** [comment]
> Elijah: Yeah, I will do that now. Need to get this knocked out.

PR is here > https://github.com/department-of-veterans-affairs/devops/pull/9604

**Comment by ElijahLynn** [comment]
@ndouglas and I debugged this further today and we were able to login to the instance and see that shortly after initialization of the instance `service httpd status` reported that `httpd` was not running. Further anaylsis revealed that the `ltime` (startup time of the parent process) was within seconds of the time that Ansible tried to restart `httpd`. See this screenshot:

![image](https://user-images.githubusercontent.com/1504756/126431242-3f970ab7-c9cb-4351-abdb-aecd7ae04147.png)

We can see that the start time of the process is 21:08:46 and the time the Ansible task ran was 21:08:39 and it failed at 21:08:48. 

So, we added an `until` task in https://github.com/department-of-veterans-affairs/devops/pull/9613 that waits for the `service httpd status` to return `is running` before doing the restart. There was an alternative way of actually updating the config before the userdata inited the `httpd` service but that was more opaque as engineers cannot see as much in the Jenkins logs then. We are still testing the PR.

**Comment by ElijahLynn** [comment]
K, so the actual issue seemed to be that the service thought it was stopped but `httpd` was still running. I tried figuring out "why" but couldn't so ended up settling on a `killall http` task just before the start. I've tested over 5 times in a row now successfully on cms-test STAGING and it would have failed by now for sure. So I think https://github.com/department-of-veterans-affairs/devops/pull/9613 is good to approve and merge. 

![image](https://user-images.githubusercontent.com/1504756/126592185-04990bf6-0aca-49a1-84c1-af0d6e397f84.png)
